### DISCUSSION TITLE
Idee

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#188] [Idee e proposte]
Buongiorno, testando il registro ci sono venute in mente delle funzionalità che potreste implementare, se possibile, nel registro:

1) possibilità di far firmare in compresenza docenti con materie diverse. Ad esempio nella mia scuola in molte ore sono in compresenza il docente di cattedra e l'ITP. Al docente di cattedra viene assegnata la materia teorica, all'ITP viene assegnata la materia di laboratorio non scrutinabile.

2) Possibilità di poter creare più classi con la stessa sezione, ma corso diverso. Ad esempio nella mia scuola abbiamo la 1AINF e la 1AGRAF, creando la 1AINF non mi permette di creare la 1AGRAF.

3) sistemare il bug riguardante la gestione degli alunni, non riesco a vederli nella area amministrativa e, essendo di aver sbagliato una lettera nel nome di un alunno, non posso correggere.

Grazie, buona giornata @trinko

**Comment by trinko** [comment]
1. il caso ITP è gestito, ma non sono ovviamente materie diverse. Si configura docente e ITP con cattedra sulla stessa materia, indicando la differenza nel tipo di cattedra (normale e ITP).

**Reply by scuolagioco** [reply]
> 1. il caso ITP è gestito, ma non sono ovviamente materie diverse. Si configura docente e ITP con cattedra sulla stessa materia, indicando la differenza nel tipo di cattedra (normale e ITP).

Ok grazie mille, per quanto riguarda gli altri 2 punti?
